examples/plugin/CMakeLists.txt

```diff
@@ -72,6 +73,22 @@ osvr_add_plugin(NAME org_osvr_example_SampleCPlugin
 set_target_properties(org_osvr_example_SampleCPlugin PROPERTIES
     FOLDER "OSVR Example Plugins")
 
+## Build the TrackerSpin example
+osvr_convert_json(com_osvr_example_TrackerSpin_json
+    com_osvr_example_TrackerSpin.json
+    "${CMAKE_CURRENT_BINARY_DIR}/com_osvr_example_TrackerSpin_json.h")
+osvr_add_plugin(com_osvr_example_TrackerSpin
+    NO_INSTALL MANUAL_LOAD CPP
```

Comment:
Can we actually get this installed, since it's useful for testing? I don't think we need manual_load here either, since it has a constructor argument.